sql/api/src/main/scala/org/apache/spark/sql/errors/ExecutionErrors.scala

```diff
@@ -120,13 +120,12 @@ private[sql] trait ExecutionErrors extends DataTypeErrorsBase {
       context: QueryContext = null): ArithmeticException = {
     val alternative = if (hint.nonEmpty) {
       s" Use '$hint' to tolerate overflow and return NULL instead."
-    } else ""
+    } else {
+      s""" If necessary set ${SqlApiConf.ANSI_ENABLED_KEY} to "false" to bypass this error."""
+    }
     new SparkArithmeticException(
       errorClass = "ARITHMETIC_OVERFLOW",
-      messageParameters = Map(
-        "message" -> message,
-        "alternative" -> alternative,
-        "config" -> toSQLConf(SqlApiConf.ANSI_ENABLED_KEY)),
+      messageParameters = Map("message" -> message, "try_alternative" -> alternative),
```

Comment:
Does `message` contain valuable info?

Reply:
Yes. It contains which type of overflow happened.